### PR TITLE
Quest - updated logos and font

### PR DESCRIPTION
**Pull request by chris-atomlearning**
The designs use Inter for all fonts, so I'm updating the theme to match. Was there a reason we were using DM sans though?
The designs use a lighter green version of the logo, so I have added that.

### CODE REVIEW

**Comment by thomasdigby** [comment]
DM Sans should be used for headings...? I thought that is what was agreed with @MiguelGDLR & Daf

File: assets/quest/logo.svg [hunk 1]
```diff
@@ -1 +1 @@
...
... 4.747-.294.177-.605.368-.914.497-.315.13-.606.292-.933.412a14.866 14.866 0 0 1-4.311.902c-1.781.102-3.711-.011-5.79-.338a36.58 36.58 0 0 1-5.693-1.358c-.028-.009-.043-.028-.044-.058l-.349-8.986c-.001-.037.017-.056.053-.056h.912c.031 0 .062.011.086.031.072.059.14.124.203.195 1.248 1.408 2.5 2.813 3.756 4.214a13.707 13.707 0 0 0 1.591 1.513c.352.287.728.54 1.128.762 1.15.636 2.402.787 3.708.708.35-.021.695-.086 1.028-.196 1-.329 1.697-.908 2.092-1.736.377-.793.437-1.646.177-2.56-.244-.86-1.267-1.608-2.056-2.01a21.819 21.819 0 0 0-2.061-.92 462.324 462.324 0 0 1-3.377-1.318c-2.106-.833-4.281-2.158-5.57-4.073-.891-1.323-1.347-2.825-1.367-4.505-.01-.617.027-1.234.109-1.845a9.229 9.229 0 0 1 1.573-4.034c.107-.156.223-.305.347-.447.177-.198.323-.422.513-.605.785-.754 1.593-1.348 2.424-1.783 1.946-1.015 4.246-1.478 6.572-1.465 1.506.008 3.051.15 4.636.426 1.528.267 3.014.623 4.455 1.068.041.009.07.046.069.088l.344 8.095c.002.03-.012.045-.042.045h-.963a.119.119 0 0 1-.093-.041 296.471 296.471 0 0 0-3.311-3.601 12.191 12.191 0 0 0-1.945-1.698c-1.062-.738-2.315-1.014-3.596-.98-1.33.036-2.588.557-3.229 1.743Z" style="fill:#185337;fill-rule:nonzero" transform="translate(-4.025 -9.743)"/></svg>
```
Comment (thomasdigby) [inline assets/quest/logo.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2" viewBox="0 0 157 48"><path d="M42.2 46.13c.05-.1.04-.11-.04-.03-3 3.15-6.55 5.2-10.68 6.14a25.4 25.4 0 0 1-6.97.59 22.12 22.12 0 0 1-12.46-4.45 20.03 20.03 0 0 1-5.82-7.2 19.78 19.78 0 0 1-.63-1.42 23.72 23.72 0 0 1-1.54-9.93c.22-3.94 1.5-7.78 3.76-10.99a20.45 20.45 0 0 1 5.8-5.55 21.67 21.67 0 0 1 2.75-1.5 23.18 23.18 0 0 1 7.76-1.97 25.4 25.4 0 0 1 7.83.6 19.7 19.7 0 0 1 3.36 1.08c.31.12.6.27.91.41a19.97 19.97 0 0 1 9.51 9.14c.19.34.33.71.5 1.07a19.86 19.86 0 0 1 1.44 4.86c.1.54.15 1.09.21 1.63.03.3.06.6.07.93a23.17 23.17 0 0 1-1.33 9.29 21.7 21.7 0 0 1-2.8 5.29c-.02.03-.02.05.01.07l8.41 5.87c.02.02.04.04.04.07a.08.08 0 0 1-.03.07l-8.83 7.33a.05.05 0 0 1-.03 0 .05.05 0 0 1-.04-.01 7.29 7.29 0 0 0-.42-.5 691.66 691.66 0 0 1-4.74-5.25.05.05 0 0 1 0-.07l.88-.95c.18-.18.34-.37.49-.57.2-.27.44-.53.64-.81a25.51 25.51 0 0 0 1.87-2.98l.13-.26Zm-13.7-.27c.72-.2 1.35-.46 1.87-.77l.3-.18c.27-.15.5-.35.73-.53.39-.3.72-.68 1.07-1.03.12-.11.18-.25.3-.37l.22-.28c.47-.66.85-1.31 1.14-1.95.7-1.52 1.13-3.12 1.4-4.77.3-2 .4-4.1.3-6.32a26.29 26.29 0 0 0-.58-4.4c-.3-1.36-.73-2.59-1.27-3.68a11.82 11.82 0 0 0-1.13-1.83c-.17-.23-.37-.43-.56-.65a6.34 6.34 0 0 0-.44-.44 8.3 8.3 0 0 0-5.2-2.2 9.13 9.13 0 0 0-3.74.43 8.52 8.52 0 0 0-4.56 3.93 13.66 13.66 0 0 0-1.22 2.82 19.8 19.8 0 0 0-.72 3.33 32 32 0 0 0 .13 8.93c.23 1.48.65 3.07 1.21 4.36a9.83 9.83 0 0 0 2.76 3.93 8.05 8.05 0 0 0 2.53 1.46c.97.33 1.98.5 3 .5a8.88 8.88 0 0 0 2.45-.29Zm113.83-22.32h-3.55c-.03 0-.04-.01-.04-.04v-.9l.01-.06a.19.19 0 0 1 .05-.06l11.92-10.92a.13.13 0 0 1 .1-.04h1.02c.03 0 .04.02.04.04v8.2c0 .02.02.03.04.03h8.02a.02.02 0 0 1 .02.02v3.69c0 .03-.02.04-.05.04H152c-.08 0-.12.04-.12.13v5.68l-.01.4a300.9 300.9 0 0 0-.02 5.54l-.02.24v4.79c.02.76.26 1.7.79 2.3.3.36.62.61.96.76.83.38 1.85.43 2.75.27a4.24 4.24 0 0 0 2.1-1c.16-.14.3-.3.42-.49a.94.94 0 0 1 .19-.2h.06l1.48.88c.02.02.03.04.02.07-.75 2.17-2 4.27-3.92 5.58a9.8 9.8 0 0 1-6.18 1.55 10.02 10.02 0 0 1-3.36-.75 7.5 7.5 0 0 1-4.22-4.3 9.84 9.84 0 0 1-.67-3.1 39.34 39.34 0 0 1-.03-2.62l.02-1.67.13-12.86c0-.39.02-.79 0-1.16 0-.03 0-.04-.03-.04ZM93.58 33.5c0 1.8.31 3.62 1.05 5.26a7.96 7.96 0 0 0 2.2 2.91 8.52 8.52 0 0 0 5.82 1.77 8.3 8.3 0 0 0 7.16-4.52c.06-.12.09-.24.15-.34.02-.03.04-.04.07-.03l2.19.82a.08.08 0 0 0 .05 0h.06c0 .15-.06.24-.1.37-.26.88-.62 1.8-1.1 2.77a14.91 14.91 0 0 1-1.94 2.99 12.34 12.34 0 0 1-5.5 3.8c-1.31.46-2.74.71-4.29.76-1.23.04-2.46-.05-3.67-.26a15.2 15.2 0 0 1-6.04-2.46 13.97 13.97 0 0 1-5.65-8.95 18.63 18.63 0 0 1-.3-4.57c.15-2.45.73-4.68 1.76-6.7a15.28 15.28 0 0 1 2.1-3.09 14.92 14.92 0 0 1 9.73-4.92 15.38 15.38 0 0 1 2.75-.02c1.22.1 2.4.34 3.56.73.34.11.67.27 1 .4.35.14.68.34 1.03.52.48.25.9.58 1.35.9.1.07.2.15.29.24.12.12.28.2.4.32.38.39.8.76 1.13 1.2.2.26.42.51.6.8.5.74.87 1.39 1.12 1.93a16.48 16.48 0 0 1 1.36 5.02c.07.67-.1 1.27-.6 1.73-.39.38-.88.57-1.46.57H93.63c-.03 0-.05.01-.05.05Zm8.39-9.14a3.72 3.72 0 0 0-3.72-1.84c-2.27.25-3.55 2.54-4.09 4.52a14.4 14.4 0 0 0-.44 2.76.07.07 0 0 0 .05.02h4.94a8.15 8.15 0 0 0 2.02-.27 2.53 2.53 0 0 0 1.8-2.1 4.7 4.7 0 0 0-.16-2.23 4.69 4.69 0 0 0-.4-.86ZM70.56 44.71c0-.06-.02-.06-.05-.01a12.96 12.96 0 0 1-2.5 2.96 9.1 9.1 0 0 1-4.85 2.22 9.98 9.98 0 0 1-7.38-1.67c-2.55-1.87-3.6-5-3.6-8.1l-.01-12.44c0-.28-.02-.57-.04-.86a3.1 3.1 0 0 0-.73-1.75l-2.15-2.77a.13.13 0 0 1-.02-.08v-.94c0-.01 0-.02.02-.02l12.56-2.16c.02 0 .04 0 .04.03V38.2c0 1.7.35 3.31 1.73 4.43.66.53 1.57.78 2.4.76.7-.01 1.45-.15 2.07-.47.27-.13.53-.25.78-.42a6.64 6.64 0 0 0 1.71-1.76.16.16 0 0 0 .03-.09V27.08c0-.78-.28-1.4-.76-2.02a517.08 517.08 0 0 0-2.13-2.76.19.19 0 0 1-.03-.11v-.92c0-.01 0-.02.02-.02l12.51-2.15c.05-.01.07 0 .07.05l.01 23.39c0 .49.09.91.25 1.27.13.27.37.61.72 1.01.76.87 1.45 1.66 2.1 2.38a.13.13 0 0 1 .03.09v.92a.06.06 0 0 1-.02.03.06.06 0 0 1-.03.02l-12.68 1.8a.06.06 0 0 1-.07-.06v-5.3Zm52.35-20.48a3.06 3.06 0 0 0-.26 2.22c.05.26.17.51.34.75s.35.44.56.6c.84.7 1.9 1.14 2.91 1.52 1.1.4 2.18.8 3.25 1.17.86.31 1.77.72 2.72 1.23 2.18 1.16 3.9 2.8 4.59 5.16.19.66.3 1.33.33 2.04a12.19 12.19 0 0 1-.2 3.02 9.14 9.14 0 0 1-4.37 6.2c-.3.19-.6.38-.92.5-.31.14-.6.3-.93.42a14.87 14.87 0 0 1-4.3.9c-1.79.1-3.72-.01-5.8-.34a36.58 36.58 0 0 1-5.7-1.36c-.02 0-.04-.02-.04-.05l-.35-8.99c0-.04.02-.06.06-.06h.91l.09.03.2.2 3.75 4.21a13.7 13.7 0 0 0 1.6 1.52 6.63 6.63 0 0 0 5.86 1.27 3.34 3.34 0 0 0 2.27-4.3c-.25-.86-1.27-1.6-2.06-2a21.82 21.82 0 0 0-2.06-.93 462.32 462.32 0 0 1-3.38-1.31c-2.1-.84-4.28-2.16-5.57-4.08a8.06 8.06 0 0 1-1.36-4.5c-.01-.62.02-1.24.1-1.85a9.23 9.23 0 0 1 1.58-4.03c.1-.16.22-.3.35-.45.17-.2.32-.42.51-.6.78-.76 1.6-1.35 2.42-1.79a14.12 14.12 0 0 1 6.58-1.46c1.5 0 3.05.15 4.63.43 1.53.26 3.02.62 4.46 1.06.04.01.07.05.07.1l.34 8.09c0 .03-.01.04-.04.04h-.97a.12.12 0 0 1-.09-.04 296.47 296.47 0 0 0-3.3-3.6 12.2 12.2 0 0 0-1.95-1.7 5.92 5.92 0 0 0-3.6-.98c-1.33.04-2.59.56-3.23 1.74Z" style="fill:#185337;fill-rule:nonzero" transform="translate(-4.03 -9.74)"/></svg>
```

**Comment by MiguelGDLR** [comment]
> 🤔 I thought what was agreed with @MiguelGDLR & Daf was that DM Sans would be used for headings, and Inter everywhere else

Figma doesn't support variables for typography yet (they are gonna release it soon), until then we can only use Inter on Figma designs. But the product should use DS styles (DM Sans).

File: assets/quest/logo.svg (hunk 1 above)
Comment (chris-atomlearning) [reply assets/quest/logo.svg:1]: Is this just with more rounding on the values?

**Comment by chris-atomlearning** [comment]
> > 🤔 I thought what was agreed with @MiguelGDLR & Daf was that DM Sans would be used for headings, and Inter everywhere else
> 
> Figma doesn't support variables for typography yet (they are gonna release it soon), until then we can only use Inter&Space grotesk on Figma designs. But the product should use DS styles (DM Sans) for headings

Ok, I can take out this change. Do we know when Figma are going to allow other fonts? It's quite confusing because in some cases we are wanting to use Inter on heading sized text, like in the banners. I guess we can just specify these though if there aren't many cases.

File: assets/quest/logo.svg (hunk 1 above)
Comment (thomasdigby) [reply assets/quest/logo.svg:1]: Mostly, there might be some other optimisations from svgo applied, but that's the main file size reduction.

File: assets/quest/logo.svg [hunk 2]
```diff
@@ -1 +1 @@
...
...2.5-4.1 4.5a14.4 14.4 0 0 0-.5 2.8 668.5 668.5 0 0 0 6.2 0l.8-.3c1-.3 1.7-1 1.8-2a4.7 4.7 0 0 0-.1-2.3 4.7 4.7 0 0 0-.4-.8ZM70.6 44.7a13 13 0 0 1-2.6 3 9 9 0 0 1-4.8 2.2 10 10 0 0 1-7.4-1.7 9.8 9.8 0 0 1-3.6-8V26.7c-.1-.6-.4-1.2-.8-1.7l-2.1-2.8v-1L61.7 19v19c0 1.8.4 3.4 1.8 4.5.6.6 1.5.8 2.4.8.7 0 1.4-.2 2-.5l.8-.4a6.6 6.6 0 0 0 1.7-1.8V27c0-.8-.2-1.4-.7-2a517 517 0 0 0-2.1-2.8.2.2 0 0 1 0-.1v-1L80.1 19v23.4c0 .5.1 1 .3 1.3l.7 1 2.1 2.4v1L70.5 50v-5.3Zm52.3-20.5a3 3 0 0 0-.3 2.3l.4.7.5.6a10 10 0 0 0 3 1.5 477.1 477.1 0 0 0 6 2.4A8.7 8.7 0 0 1 137 37l.4 2a12.2 12.2 0 0 1-.3 3c0 .5-.2 1-.4 1.5-.7 2-2 3.6-4 4.7l-.8.5-1 .5a14.9 14.9 0 0 1-4.3.9c-1.8 0-3.7 0-5.8-.4a36.6 36.6 0 0 1-5.7-1.3l-.4-9 .1-.1h1l.2.2 3.8 4.2a13.7 13.7 0 0 0 1.5 1.5l1.2.8a6.6 6.6 0 0 0 4.7.5 3.3 3.3 0 0 0 2.3-4.3c-.3-.9-1.3-1.6-2-2a21.8 21.8 0 0 0-2.1-1 462.3 462.3 0 0 1-3.4-1.3 12 12 0 0 1-5.6-4 8 8 0 0 1-1.4-4.5c0-.6 0-1.3.2-1.9a9.2 9.2 0 0 1 1.5-4l.4-.5.5-.6c.8-.7 1.6-1.3 2.4-1.7 2-1 4.3-1.5 6.6-1.5a27.9 27.9 0 0 1 9.1 1.6l.4 8v.1h-1a.1.1 0 0 1-.1 0 296.5 296.5 0 0 0-3.3-3.6 12.2 12.2 0 0 0-2-1.7 6 6 0 0 0-3.6-1c-1.3 0-2.5.5-3.2 1.7Z" style="fill:#185337;fill-rule:nonzero" transform="translate(-4 -9.7)"/></svg>
```
Comment (thomasdigby) [inline assets/quest/logo.svg:1]: I know this is over optimising, but as the change to this logo is only the colour, maybe we can just use the previous logo and update the `fill`?

Comment (chris-atomlearning) [reply assets/quest/logo.svg]: Good point, that one does look neater.

**Review by thomasdigby — APPROVED** [review]

**Pull request merged**